yt/frontends/swift/tests/test_outputs.py

```diff
@@ -15,7 +15,7 @@
 @requires_file(keplerian_ring)
 def test_non_cosmo_dataset():
     ds = load(keplerian_ring)
-    assert(type(ds) == SwiftDataset)
+    assert(type(ds) is SwiftDataset)
```

Comment:
```suggestion
    assert(isinstance(ds, SwiftDataset)
```

Reply:
same as above, not willing to make tests more flexible here.